src/Terrain/terrain.cpp

```diff
@@ -116,102 +144,128 @@ void Terrain::init(int x, int y, int Area_size_, int z, int seed_, const std::ma
     std::map<int, LandGenerator> land_generators;
 
     // for tile macro in data biome
-    for (unsigned int i = 0; i < biome_data["Tile_Macros"].size(); i++){
+    for (unsigned int i = 0; i < biome_data["Tile_Macros"].size(); i++) {
         // create a land generator for each tile macro
         LandGenerator gen(materials, biome_data["Tile_Macros"][i]["Land_Data"]);
         land_generators.insert(std::make_pair(i, gen));
     }
 
     for (int i = 0; i < x; i++)
-    for (int j = 0; j < y; j++){
-        for (unsigned int k = 0; k < biome_data["Tile_Data"][Terrain_Maps[j+i*y]]["Land_From"].size(); k ++){
-            init_area(i, j, land_generators[biome_data["Tile_Data"][Terrain_Maps[j+i*y]]["Land_From"][k].asInt()]);
+        for (int j = 0; j < y; j++) {
+            for (unsigned int k = 0;
+                 k <
+                 biome_data["Tile_Data"][Terrain_Maps[j + i * y]]["Land_From"]
+                     .size();
```

Comment:
```suggestion
                 k < biome_data["Tile_Data"][Terrain_Maps[j + i * y]]["Land_From"].size();
```
WHAT was that indentation

Reply:
don't worrie about it